packages/main/src/components/AnalyticalTable/virtualization/VirtualTableBody.tsx

```diff
@@ -40,7 +39,7 @@ export const VirtualTableBody = (props) => {
         columns
       }
     };
-  }, [rows, prepareRow, isTreeTable, classes, columns, selectedFlatRows, selectionMode]);
```

Comment:
as far as I remember, those dependencies are needed due to the memoization of the selected rows but I'm not 100% sure